upgrade/archive-status.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+update_schema() 
+{
+curl -k $elastic_auth -X PUT "$elastic_url/registry/_mapping" -H 'Content-Type: application/json' -d'
```

Comment:
My only comment is that—since code is _read_ more often than it is _writtern_—that I like to see `--full-command-options` to make understanding easier:
```
curl --insecure "$elastic_auth" --request PUT --header 'Content-type: application/json' "$elastic_url/registry/_mapping" …
```
Otherwise looks fine.